core/src/main/java/com/netflix/conductor/dao/MetadataDAO.java

```diff
@@ -1,41 +1,34 @@
-/**
- * Copyright 2016 Netflix, Inc.
+/*
+ * Copyright 2019 Netflix, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/**
- *
- */
 package com.netflix.conductor.dao;
 
-import com.netflix.conductor.common.metadata.events.EventHandler;
 import com.netflix.conductor.common.metadata.tasks.TaskDef;
 import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
-
 import java.util.List;
 import java.util.Optional;
 
 /**
- * @author Viren
  * Data access layer for the workflow metadata - task definitions and workflow definitions
  */
 public interface MetadataDAO {
 
     /**
      * @param taskDef task definition to be created
-     * @return name of the task definition
      */
-    String createTaskDef(TaskDef taskDef);
+    void createTaskDef(TaskDef taskDef);
```

Comment:
Instead of void, should it return something confirming that TaskDef is actually created?

Reply:
This is a synchronous request from the API so if the creation fails, the request would receive a non-200 status code.